### ISSUE TITLE
Can't access note screen

### ISSUE THREAD

**Issue by DiaLange** [shuowu/yi-note#92]
When I click on the Yinote extension to take notes on a video, it just opens teh Yinote management page, no matter what I do. I can't get it to open the notes screen to take notes. I am using a macbook air using High Sierra. 
![Screen Shot 2021-05-02 at 11 25 19 AM](https://user-images.githubusercontent.com/83548425/116820151-1e9ca500-ab39-11eb-8e9c-5a1ccaa01703.png)

**Comment by sahilgupta8911** [comment]
Hey, were you able to resolve?

**Comment by DiaLange** [comment]
No. Am I doing something wrong? I have a youtube video opened, and then I click on the extension. Is there something else I'm supposed to do? I watched someone demonstrate it online and that is all they did, but when I do that, I just get the Yinote management page as shown in my picture.

**Comment by thomasvs** [comment]
I have the same issue.  Always opens the settings page, never the frame on the right in the same page as the video.

**Comment by DiaLange** [comment]
Does anyone have a solution?

**Comment by KeaganQa** [comment]
Hello,
I had the same problem and I did follow a youtube video talking on how to use Yi note, I'm not sure it's this but it worked for me. 
When you go into the setting you need :

- Preset second to seek : 0  ( if you select another you need the video paused to be at the time you selected I think)

- Apply preset seconds to seek to generated URLs (URLs in generated PDF, notes sent to third party platforms) : Off

- Pause video when editing note : On (maybe you can set this Off and be able to directly open the extention without having to pause the video at the beginning) 

- Relaod tab when URL change : Off 

 You might just need to put the video on 0:00 and pause it and then click on the extension for Yinote side bar to appear. 

Hope this will help you ! 

Good luck and Take care guys !
